modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -826,7 +827,7 @@ object SeqexecEngine extends SeqexecConfiguration {
   private val refreshSequences: Endo[EngineState] = (st:EngineState) => {
     st.sequences.map{ case (id, obsseq) => updateSequenceEndo(id, obsseq) }.foldLeft(st){case (s, f) => f(s)}
   }
-
+  private val logger = Logger.getLogger(this.getClass.getName)
```

Comment:
Do we want to keep this?

Reply:
No... I added a bunch of loggers and will get rid of them all, along with the log messages. This was just to help me see the flow of the code and see what code was being executed.